.github/workflows/release-beta.yml

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: release-beta
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'Release type: Major; Minor; Patch'
+        required: true
+permissions: read-all
+jobs:
+  Beta:
+    permissions:
+      contents: write
+      packages: write
+    runs-on: ubuntu-latest
+    env:
+      COSIGN_KEY_LOCATION: /tmp/cosign.key
+      COSIGN_PWD: ${{ secrets.COSIGN_PWD }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Docker Login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Cosign Installer
+        uses: sigstore/cosign-installer@main
+        with:
+          cosign-release: 'v1.2.0'
+
+      - name: Install Mage
+        run: go install github.com/magefile/mage@v1.11
```

Comment:
The mage's last release was almost [a year ago](https://github.com/magefile/mage/releases/tag/v1.11.0), are we safe with that?

Reply:
the maintainers dropped this [discussion](https://github.com/magefile/mage/discussions/369) not so long ago on the project, i think i could help him with some issues/prs. It's used in k8s release porcess, hugo and logrus, i don't think it will fail or be unamaintened, we always can change to bash scripting if something doesn't match our expectations.